demos/pedestrian_tracker_demo/src/detector.cpp

```diff
@@ -99,20 +106,37 @@ ObjectDetector::ObjectDetector(
     }
 
     InputsDataMap inputInfo(net_reader.getNetwork().getInputsInfo());
-    if (inputInfo.size() != 1) {
-        THROW_IE_EXCEPTION << "Person Detection network should have only one input";
+    switch(inputInfo.size()) {
+        case 1: {
+            InputInfo::Ptr inputInfoFirst = inputInfo.begin()->second;
+            inputInfoFirst->setPrecision(Precision::U8);
+            inputInfoFirst->getInputData()->setLayout(Layout::NCHW);
+            input_name_ = inputInfo.begin()->first;
+            im_info_name_.clear();
+            break;
+        }
+        case 2: {
```

Comment:
Doesn't the `case 2` code also work for `case 1`?

Reply:
Updated